docs/detections/rules-coverage.asciidoc

```diff
@@ -1,9 +1,51 @@
 [[rules-coverage]]
 == MITRE ATT&CK® coverage
 
-:frontmatter-description: 
+:frontmatter-description: Review your current coverage of MITRE ATT&CK® tactics and techniques, based on installed rules.
 :frontmatter-tags-products: [security]
 :frontmatter-tags-content-type: [how-to]
-:frontmatter-tags-user-goals: [manage]
+:frontmatter-tags-user-goals: [manage, analyze, visualize]
 
-This is a placeholder for future documentation.
+The **MITRE ATT&CK® coverage** page shows which https://attack.mitre.org[MITRE ATT&CK®] adversary tactics and techniques are covered by your installed and enabled detection rules. This includes both Elastic prebuilt rules and custom rules.
+
+Mirroring the MITRE ATT&CK® framework, columns represent major tactics, and cells within each column represent a tactic's related techniques. Cells are darker when a technique has more rules matching the current filters, as indicated in the **Legend** at the top.
+
+[NOTE]
+====
+This page only includes the detection rules you currently have installed, and only rules that are mapped to MITRE ATT&CK® tactics. Additionally, rules must be mapped to the MITRE ATT&CK® version used by {elastic-sec}: `v12.1`. If there are Elastic prebuilt rules you haven't installed yet, or custom rules that you either haven't mapped or are mapped to a deprecated tactic or technique, they will not be included in determining your MITRE ATT&CK® coverage.
```

Comment:
```suggestion
This page only includes the detection rules you currently have installed, and only rules that are mapped to MITRE ATT&CK® tactics. Additionally, rules must be mapped to the MITRE ATT&CK® version used by {elastic-sec}: `v12.1`. Elastic prebuilt rules that are not installed and custom rules that are either unmapped or mapped to a deprecated tactic or technique will not be included.
```

Reply:
Expert edits for conciseness -- thanks @benironside!